src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -51,15 +51,16 @@
 		<span v-if="isLoading"
 			v-tooltip="previewTooltip"
 			class="preview loading" />
-		<button v-if="isUploadEditor"
+		<Button v-if="isUploadEditor"
 			tabindex="1"
+			type="primary"
 			:aria-label="removeAriaLabel"
-			class="remove-file primary">
+			class="remove-file">
 			<Close class="remove-file__icon"
 				decorative
 				title=""
 				@click="$emit('remove-file', id)" />
```

Comment:
Use the icon slot like in:
https://github.com/nextcloud/spreed/pull/6980/files#r823558362